bigframes/bigquery/utils.py

```diff
@@ -0,0 +1,85 @@
+# Copyright 2024 Google LLC
```

Comment:
No `utils` files, please (go/no-utils). We should consolidate our SQL generation following Trevor's recent refactor. See: https://github.com/googleapis/python-bigquery-dataframes/pull/682

Reply:
Done.